lib/config/options/index.ts

```diff
@@ -2611,6 +2611,17 @@ const options: RenovateOptions[] = [
     default: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'gerritLabelMapping',
+    description:
+      'The name of the Gerrit labels to use for the `StabilityDays` and `Merge-Confidence` (branch)status.',
+    type: 'object',
+    default: {
+      stabilityDaysLabel: undefined,
```

Comment:
`stabilityDays` is now `minimumReleaseAge`. Please update the docs and code. 😉 

https://docs.renovatebot.com/configuration-options/#minimumreleaseage